lib/modules/versioning/docker/index.spec.ts

```diff
@@ -1,3 +1,4 @@
+import maven from '../maven';
```

Comment:
nope. don't write maven versioning tests at docker versioning

Reply:
I know, this is not a pull for merging, but rather for demo purpose.
I would rather see that i can hint the engine that a different versioning is used, aka use maven versioning.

Issue: https://github.com/renovatebot/renovate/discussions/18498